precompiles/avs/abi.json

```diff
@@ -56,12 +178,65 @@
 	{
 		"inputs": [
 			{
-				"internalType": "uint64",
-				"name": "action",
-				"type": "uint64"
+				"internalType": "string",
+				"name": "operator",
+				"type": "string"
+			}
+		],
+		"name": "getRegisteredPubkey",
+		"outputs": [
+			{
+				"internalType": "bytes",
+				"name": "pubkey",
+				"type": "bytes"
+			}
+		],
+		"stateMutability": "nonpayable",
+		"type": "function"
+	},
+	{
+		"inputs": [
+			{
+				"internalType": "string",
+				"name": "TaskContractAddress",
+				"type": "string"
+			},
+			{
+				"internalType": "string",
+				"name": "Name",
+				"type": "string"
+			},
+			{
+				"internalType": "string",
+				"name": "MetaInfo",
+				"type": "string"
+			}
+		],
+		"name": "registerAVSTask",
+		"outputs": [
+			{
+				"internalType": "bool",
+				"name": "success",
+				"type": "bool"
+			}
+		],
+		"stateMutability": "nonpayable",
+		"type": "function"
+	},
+	{
+		"inputs": [
+			{
+				"internalType": "string",
+				"name": "operator",
+				"type": "string"
+			},
+			{
+				"internalType": "bytes",
+				"name": "pubKey",
+				"type": "bytes"
 			}
 		],
-		"name": "OperatorOptAction",
+		"name": "registerBLSPublicKey",
```

Comment:
Should also support deregiste?

Reply:
registerBLSPublicKey ?  why your function name  sometimes starting with uppercase, sometimes lowercase ?